### ISSUE TITLE
Problems parsing JSON

### ISSUE THREAD

**Issue by AdrienLemaire** [kaizensoze/node-github#100]
I am unable to add labels to a pull request using this api.

Query:

```
params = {                                                                                                                                                                                                                               
  user: process.env.GH_USER,                                                                                                                                                                                                             
  repo: process.env.GH_REPO,                                                                                                                                                                                                             
  number: 6,                                                                                                                                                                                                                             
  body: ["auto"]                                                                                                                                                                                                                         
};                                                                                                                                                                                                                                       
github.issues.addLabels(params, function(err, labels) {                                                                                                                                                                                  
  if (err) { console.log(JSON.stringify(err)); }                                                                                                                                                                                         
  else {                                                                                                                                                                                                                                 
    console.log(prettyjson.render(labels));                                                                                                                                                                                              
  }                                                                                                                                                                                                                                      
}); 
```

Result

```
[error] { [Error: {"message":"Problems parsing JSON","documentation_url":"https://developer.github.com/v3/issues/labels/#add-labels-to-an-issue"}]
```

**Comment by AdrienLemaire** [comment]
The problem seems to come from the request sending `{"body": ["auto"]}` to the server instead of just `["auto"]`

**Comment by kaizensoze** [comment]
This should be fixed by #98 and is included in the latest 0.5.5.
